dist/profile/templates/confluence/vhost.conf

```
@@ -2853,12 +2853,86 @@ RewriteRule "^/display/JENKINS/Subversion\+Plugin$" "https://plugins.jenkins.io/
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Kubernetes\+Plugin$" "https://plugins.jenkins.io/kubernetes" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
-RewriteRule "^/display/JENKINS/Warnings\+Next\+Generation+Plugin$" "https://plugins.jenkins.io/warnings-ng" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Warnings\+Next\+Generation\+Plugin$" "https://plugins.jenkins.io/warnings-ng" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Lockable\+Resources\+Plugin$" "https://plugins.jenkins.io/lockable-resources" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Warnings\+Plugin$" "https://plugins.jenkins.io/warnings" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^display/JENKINS/Bitbucket\+Branch\+Source\+Plugin$" "https://plugins.jenkins.io/cloudbees-bitbucket-branch-source" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Splunk\+Plugin\+for\+Jenkins$" "https://plugins.jenkins.io/pmd" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/PMD\+Plugin$" "https://plugins.jenkins.io/splunk-devops" [NC,L,QSA,R=301]
```

PMD plugin is end of life, replaced by warnings-ng.
```suggestion
RewriteRule "^/display/JENKINS/PMD\+Plugin$" "https://plugins.jenkins.io/warnings-ng" [NC,L,QSA,R=301]
```

---

Shouldn't the docs still go to PMD? The important information for users is that the plugin is derecated and what to use instead (as mentioned in https://plugins.jenkins.io/pmd now). Showing documentation for another plugin is IMHO confusing.

---

I think you're right @zbynek.  I've modified my proposal.

---

Fixed higher